Classes/Core/Runtime/FormRuntime.php

```diff
@@ -181,7 +182,7 @@ protected function initializeFormStateFromRequest()
             $this->formState = new FormState();
         } else {
             $serializedFormState = $this->hashService->validateAndStripHmac($serializedFormStateWithHmac);
-            $this->formState = unserialize(base64_decode($serializedFormState), ['allowed_classes' => [FormState::class]]);
+            $this->formState = unserialize(base64_decode($serializedFormState), ['allowed_classes' => [FormState::class, DateTime::class]]);
```

Comment:
Does this work as well:
```suggestion
            $this->formState = unserialize(base64_decode($serializedFormState), ['allowed_classes' => [FormState::class, \DateTimeInterface::class]]);
```
?

Reply:
No. Unfortunately not.